lntest/mock/walletcontroller.go

```diff
@@ -181,7 +181,8 @@ func (w *WalletController) ListUnspentWitness(int32, int32,
 
 // ListTransactionDetails currently returns dummy values.
 func (w *WalletController) ListTransactionDetails(int32, int32,
-	string) ([]*lnwallet.TransactionDetail, error) {
+	string, func(*lnwallet.TransactionDetail) bool,
+) ([]*lnwallet.TransactionDetail, error) {
```

Comment:
nit: can we format this the same way as in `lnwallet/mock.go` please?

Reply:
Done